Packs/ReversingLabs_Titanium_Cloud/ReleaseNotes/2_7_1.md

```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### ReversingLabs TitaniumCloud v2
+- Updated the ***url*** command:
+  - Updated the human-readable response it can exclude the "Statistics" section if it is not present in the API response.
```

Comment:
```suggestion
- Updated the human-readable response in the ***url*** command so that the "Statistics" section can be excluded if it is not present in the API response.
```
To fit XSAOR standards